fdbbackup/backup.actor.cpp

```diff
@@ -2219,9 +2218,26 @@ ACTOR Future<Void> runFastRestoreAgent(Database db, std::string tagName, std::st
 				dbVersion = desc.maxRestorableVersion.get();
 				TraceEvent("FastRestoreAgent").detail("TargetRestoreVersion", dbVersion);
 			}
-			Version _restoreVersion = wait(fastRestore(db, KeyRef(tagName), KeyRef(container), waitForDone, dbVersion,
-			                                           verbose, range, KeyRef(addPrefix), KeyRef(removePrefix)));
-			restoreVersion = _restoreVersion;
+			state UID randomUID = deterministicRandom()->randomUniqueID();
+			TraceEvent("FastRestoreAgent")
+			    .detail("SubmitRestoreRequests", ranges.size())
+			    .detail("RestoreUID", randomUID);
+			wait(backupAgent.submitParallelRestore(db, KeyRef(tagName), ranges, KeyRef(container), dbVersion, true,
```

Comment:
This `backupAgent.submitParallelRestore ` is tested by simulation while the deleted `fastRestore()` does not. 
The `fastRestore()` stops working without being noticed, which caused pain when I run circus test. :(